src/crystal/event_loop.cr

```diff
@@ -21,8 +21,8 @@ abstract class Crystal::EventLoop
   end
 
   # Creates an event loop instance
```

Comment:
```suggestion
  # Creates an event loop instance.
  #
  # The *parallelism* arg is informational. It reports how many schedulers are
  # expected to register with the event loop instance. Because schedulers are
  # dynamically started and execution contexts can be resized, more or less
  # schedulers may really register in practice.
```